scripts/submit_form.js

```diff
@@ -0,0 +1,113 @@
+// Google Apps Script to submit a form to a Google Sheet and send an email notification
+
+var SCRIPT_PROPERTIES = PropertiesService.getScriptProperties();
+const EMAIL_ADDRESS = "";  // set this!
+
+const formIdToNameMap = new Map([
+  ["feedback", "Give Feedback"],
+  ["report_missing_info", "Report Missing or Outdated Information"],
+  ["partner_with_freefrom", "Partner with FreeFrom"],
+  ["build_collective_survivor_power", "Build Collective Survivor Power"],
+  ["policy_ideas", "Share your Policy Ideas"],
+]);
+
+function setup() {
+  var doc = SpreadsheetApp.getActiveSpreadsheet();
+  SCRIPT_PROPERTIES.setProperty("key", doc.getId());
+}
+
+function doPost(request) {
+  const { _, postData: { contents, type } = {} } = request;
+
+  if (type !== "application/json") {
+    return ContentService.createTextOutput(
+      JSON.stringify({
+        result: "error",
+        error: "expected type application/json",
+      })
+    ).setMimeType(ContentService.MimeType.JSON);
+  }
+
+  const data = JSON.parse(contents);
+  return submitForm(flattenArrays(data));
+}
+
+// turns arrays into a string of comma-separated values
+function flattenArrays(data) {
+  for (var key in data) {
+    if (Array.isArray(data[key])) {
+      data[key] = data[key].join(", ");
+    }
+  }
+  return data;
+}
+
+function submitForm(data) {
+  var lock = LockService.getScriptLock();
+  lock.waitLock(10000); // wait 10 seconds before conceding defeat
+
+  try {
+    var doc = SpreadsheetApp.openById(SCRIPT_PROPERTIES.getProperty("key"));
+    var formId = data["form"];
+    var formName = formIdToNameMap.get(formId);
+    var sheet = doc.getSheetByName(formName);
```

Comment:
We'll ensure the Flask app always passes in a `form`, which should map to a sheet name in the Google Sheet